pkg/api/types.go

```diff
@@ -334,6 +467,8 @@ type Capabilities struct {
 type ResourceRequirements struct {
 	// Limits describes the maximum amount of compute resources required.
 	Limits ResourceList `json:"limits,omitempty"`
+	// Requests describes the minimum amount of compute resources required.
+	Requests ResourceList `json:"requests,omitempty"`
```

Comment:
Perhaps you and @bgrant0607 could hash this out?  This was his feedback.  I can use the existing Limit field and know the correct usage when matching the claim to a pv.